ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DagUtils.java

```diff
@@ -1811,4 +1819,11 @@ static long parseRightmostXmx(String javaOpts) {
     }
     return allNonAppFileResources;
   }
+
+  public static void initTezAttributes(Configuration conf, ProcessorContext context) {
```

Comment:
Move this in TezProcessor itself? It is not DAG specific and is needed mainly for logging purposes. Having it within TezProcessor will make it easier to read and not confuse anyone trying to walk through DagUtils.

Reply:
makes sense
I want to rely on this in later patches, e.g. HIVE-24278, but I think we can still make use of these properties runtime if it's moved to  TezProcessor, I'll do that